enterprise/j2ee.dd/external/binaries-list

```diff
@@ -1,3 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
 D2F2B34B7D0A13256D491B14590ED23CA0E49B46 javaee-api-5.jar
 2801A0BB60C19040FDA8C4801963911255774718 j2ee.dd-glassfish-resources.zip
 12763FC339B86791D848078A479793C3FF815982 j2ee.dd-w3c-resources.zip
```

Comment:
It would be great if you could check if you can remove the use of the custom download server. The javaee-api should be available on maven central, for the other two resources, you could see if you can adapt the methods Reema and Sarvesh  applied:

https://github.com/apache/incubator-netbeans/pull/935
https://github.com/apache/incubator-netbeans/pull/877

If you feel uncomfortable with that, please indicate so, that I'll take a look at it.

Reply:
@matthiasblaesing To resolve the ZIP external dependencies, can I find the files contained within the ZIP and add them to the project...of course adding the appropriate license?  For instance, with the w3c-resources, there are three files within that archive.  Can I simply copy those files into the appropriate location within the j2ee.dd module structure, or do I need to obtain those files from maven central somewhere?  Thanks

Reply:
It depends. For XSDs + DTDs it can be argued, that they are used at runtime and thus need to be included in their source form. We are currently integrating them if possible (apache legal faq describes this as Category B). The referenced PRs take that road.

For the javaee-api-5 dependency, this should be switch to the maven dependency `javaee:javaee-api:5`